### PR TITLE
Ian's OMERO image chooser

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/YhnO4YnX/459-ian-s-omero-imagechooser

### CODE REVIEW

**Comment by imunro** [comment]
Looks great. Thanks Helen.

File: _posts/2016-08-22-image-chooser.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+It is a graphical tool which allows a variety of OMERO objects (Images,
+Datasets, Plates, Attachments) to be selected with a look and feel modelled on
+OMERO.insight. It can easily be called from Java or Matlab code and is
+presented as a quick and relatively easy path to either developing a new OMERO
+desktop  client or adding an OMERO interface to existing code. 
+
+The source is available from [https://github.com/imperial-photonics/omeUiUtils](https://github.com/imperial-photonics/omeUiUtils)
+
+A ready-to-use .jar file compatible with OMERO 5.2.x  can be also  downloaded
+from [https://bintray.com/imperial-photonics/omeUiUtils/omeUiUtils/0.1.5#files](https://bintray.com/imperial-photonics/omeUiUtils/omeUiUtils/0.1.5#files)
```
Comment (joshmoore) [inline _posts/2016-08-22-image-chooser.md]: Perhaps use https://bintray.com/imperial-photonics/omeUiUtils/omeUiUtils/_latestVersion ?

Comment (hflynn) [inline _posts/2016-08-22-image-chooser.md]: Oh, does that automatically work? Very handy!

Comment (imunro) [inline _posts/2016-08-22-image-chooser.md]: Indeed! Did not know that

**Comment by imunro** [comment]
Re: @sbesson 's comment that it would be good to include a description of how to use this from Maven. I just built a version of ThunderSTORM using the new bintray setup and that seemed to work fine. However it didn't seem quite as simple as dropping the text from https://bintray.com/imperial-photonics/omeUiUtils/omeUiUtils/0.1.5 & "set me up" into the pom.  @joshmoore or Seb I don't suppose you have the time to contribute a brief description of the procedure.  Obviously I worked it out at some point in the past but it's gone now.

**Comment by sbesson** [comment]
@imunro: is the code linking this utility JAR into ThunderSTORM available publicly? If so, linking to a concrete integration example of usage might be largely sufficient as a starting point.

**Comment by imunro** [comment]
https://github.com/imunro/thunderstorm/blob/OME-master/pom.xml Seems to work.

**Comment by sbesson** [comment]
For instructions on how to integrate with Maven/Gradle, we could link to https://bintray.com/docs/usermanual/formats/formats_mavenrepositories.html#_working_with_maven.

**Comment by imunro** [comment]
@sbesson make it someone els's problem - Love it! Sounds ideal

**Comment by imunro** [comment]
Looks good to me but I don't know Gradle from a hole in the ground. :-)

File: _posts/2016-08-23-image-chooser.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+presented as a quick and relatively easy path to either developing a new OMERO
+desktop  client or adding an OMERO interface to existing code. 
+
+The source is available from [https://github.com/imperial-photonics/omeUiUtils](https://github.com/imperial-photonics/omeUiUtils).
+
+A ready-to-use .jar file compatible with OMERO 5.2.x  can be also  downloaded
+from [https://bintray.com/imperial-photonics/omeUiUtils/omeUiUtils/](https://bintray.com/imperial-photonics/omeUiUtils/omeUiUtils/_latestVersion).
+
+For instructions on how to integrate with Maven/Gradle, refer to the
+[bintray user documentation](https://bintray.com/docs/usermanual/formats/formats_mavenrepositories.html#_working_with_maven).
```
Comment (sbesson) [inline _posts/2016-08-23-image-chooser.md]: Bintray

**Comment by sbesson** [comment]
Looks good to me.

**Comment by imunro** [comment]
Many thanks Guys! Much appreciated.

**Pull request merged**